storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValues.java

```diff
@@ -92,8 +94,30 @@ public Builder(@NonNull StorIOContentResolver storIOContentResolver, @NonNull Co
          * @return builder
          */
         @NonNull
-        public Builder withPutResolver(@NonNull PutResolver<ContentValues> putResolver) {
+        public CompleteBuilder withPutResolver(@NonNull PutResolver<ContentValues> putResolver) {
             this.putResolver = putResolver;
+            return new CompleteBuilder(this);
+        }
+    }
+
+    /**
+     * Builder for {@link PreparedPutContentValues}
+     */
+    public static class CompleteBuilder extends Builder {
```

Comment:
I think, `CompleteBuilder` should override all `Builder` methods and change return type to `CompleteBuilder`?